requirements.txt

```diff
@@ -1,4 +1,4 @@
-Click>=7.0,!=8.0.0
+Click>8.0.0
```

Comment:
should we just try?
```suggestion
Click!=8.0.0
```

Reply:
With this PR, I just wanted to check if anything breaks if we require>8.0.0. Nothing seems to break so I'd close the issue. 
We do need the lower bound though, see https://click.palletsprojects.com/en/8.0.x/upgrading/#upgrading-to-7-0 (<7.0 would likely need special treatment).

Reply:
ah got it about >7.0 what about `==8.0.0`?

Reply:
ah you are already did it... funny

Reply:
@dannygoldstein figured this out a long time ago :)
https://github.com/wandb/client/pull/2261

Reply:
ah should we leave a comment for future investigations?